lumigator/rayserve/serve.py

```diff
@@ -0,0 +1,151 @@
+import logging
```

Comment:
Is this on an import path that will get included in lumigator code?

Reply:
Sorry, I don't get that. This script will get pulled by Ray and executed in the Python environment you specify (e.g., the one we `pip install vllm`) just like any other Python script. It does not have any dependency to Lumigator.

Reply:
We have a bit of a non-standard way of including code in the main package See [here](https://github.com/mozilla-ai/lumigator/blob/main/Dockerfile#L8) but looks like this won't get included there. VLLM is not a portable import, it doesn't work on mac, so we need to be careful where this code is getting included (automatically or otherwise)